compechem/calculators/calculators.py

```diff
@@ -0,0 +1,276 @@
+import os, shutil
+from rdkit import Chem
+from tempfile import mkdtemp
+
+parent_dir = os.getcwd()
+
+
+class Molecule:
+    def __init__(self, xyz_file, charge=0, spin=1) -> None:
+        self.moleculename: str
+        self.charge: int = charge
+        self.spin: int = spin
+        self.atomcount: int
+        self.geometry: list = []
+        # Energies
+        self.energies = {
+            "total_energy": 0.0,
+            "vibronic_contribution": 0.0,
+            "spe_energy": 0.0,
+        }
```

Comment:
Implementerei come dataclass!
```suggestion
from dataclasses import dataclass
@dataclass
class MoleculeEnergies:
     total_energy: float
     vibronic_contribution: float
     spe_energy: float

class Molecule:
    def __init__(self, xyz_file, charge=0, spin=1) -> None:
    ...
    self._energies = None
    
    @property
    def energies(self):
        ...

    @energies.setter
    def energies(self, value):
        ...
```

Metterei `None` come sentinel value per far capire che l'oggetto è solo stato inizializzato ma non è mai stato eseguito alcun calcolo. 

Poi i vari metodi di calcolo possono scrivere un oggetto `MoleculeEnergies` e aggiornare quello esistente in `Molecule`. `Molecule.energies` poi può venire gestito con delle properties :)

Thoughts?

Reply:
`def __init__(self, xyz_file, charge=0, spin=1) -> None:` aggiunto

Per il discorso energie, dobbiamo pensarci bene, non sono convinto del "dizionario" coi vari metodi...